pages/pipelines/input_step.md

```diff
@@ -275,3 +275,22 @@ steps:
             value: "stable"
 ```
 {: codeblock-file="pipeline.yml"}
+
+## Input validation
+
+To prevent the users from entering invalid values when using an input step with text inputs (for example, to gather some deployment information), you can use input validation.
```

Comment:
I think this could be a little tighter, but happy for you to leave it as-is if you think it's clearer:

```suggestion
To prevent users from entering invalid text values in input steps  (for example, to gather some deployment information), you can use input validation.
```